roster2/bundle/src/bundle/card_game_eu.properties

```diff
@@ -16,4 +16,7 @@ sound_enable = Gaitu soinu efektuak
 sound_disable = Desgaitu soinu efektuak
 user_image_alt = Asmatu behar duzun ikaslearen argazkia
 user_name_select_placeholder = Sartu ikaslearen izena...
-whats_the_user_name = Zein da ikaslearen izena?
\ No newline at end of file
+whats_the_user_name = Zein da ikaslearen izena?
+user_progress = {0} of {1} attempts were correct for this student.
```

Comment:
Hi @PabloSanRoman!
Could you please help me with those translations?
Here are the English versions:
```
user_progress = {0} of {1} attempts were correct for this student.
mark_learned = I know this student, don\u2019t show it to me in this round.
no_name_found = No name found for this input.
```

Greetings,
Markus